host-configs/tpls.cmake

```diff
@@ -52,6 +52,10 @@ if(EXISTS ${GEOSX_TPL_DIR}/fesapi)
   set(FESAPI_DIR ${GEOSX_TPL_DIR}/fesapi CACHE PATH "" FORCE)
 endif()
 
+if(EXISTS ${GEOSX_TPL_DIR}/grpc)
+  set(GRPC_DIR ${GEOSX_TPL_DIR}/grpc CACHE PATH "" FORCE)
+endif()
```

Comment:
@MichaelSekachev This needs to be the correct path to grpc.  I actually don't know the correct path, so you should correct it if this is incorrect.

Reply:
Thank you. Apparently, I did not see where I am supposed to do it. I was distracted by a variable name the same way in the main CMakeLists.txt file of the TPL itself. Thank you for going the extra mile and pushing the commit. I was able to configure the code just fine now. Next step is to play with cpp files and try to add an include file in one of the cpp files. I just randomly picked `src/coreComponents/functions/CompositeFunction.hpp` and I think I need to modify the `src/coreComponents/functions/CMakeLists.txt` similar to MATHPRESSO.